charts/posthog/templates/migrate.job.yaml

```diff
@@ -33,16 +33,12 @@ spec:
       containers:
       - name: migrate-job
         image: {{ template "posthog.image.fullPath" . }}
-        # switch over to using /bin/migrate once 1.34 is released - https://github.com/PostHog/posthog/pull/8872
         command:
           - /bin/sh
           - -c
           - |
-            set -e
-            python manage.py notify_helm_install || true
-            python manage.py migrate
-            python manage.py migrate_clickhouse
-            python manage.py run_async_migrations --check
```

Comment:
@tiina303 talking about code drift, I see we weren't running `python manage.py sync_replicated_schema`. Is that a problem?